client-js/app/elements/labrad-plot.ts

```diff
@@ -4,6 +4,9 @@ import {viridisData} from '../scripts/colormaps';
 import * as datavault from "../scripts/datavault";
 
 
+const MOUSE_LEFT_BUTTON = 0;
```

Comment:
Maybe call this `MOUSE_MAIN_BUTTON` with a comment that the main button is usually the left, but users can configure this (this aligns with [mozilla's docs](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button)).

I gotta say, it's annoying that the web apis have so many magic numbers and don't define standard symbolic names for these kind of things, but such is life.